adhocracy-plus/assets/js/unload_warning.js

```diff
@@ -26,11 +28,11 @@ $(function () {
   //   })
   // }
```

Comment:
Now the embed is gone can we add this function back in?

Reply:
technically the embed isn't gone yet, is it?

Reply:
uncommented it, seems to work